js/src/bitget.js

```diff
@@ -825,6 +825,7 @@ export default class bitget extends Exchange {
                     '40017': ExchangeError,
                     '40018': PermissionDenied,
                     '40019': BadRequest,
+                    '40037': AuthenticationError,
```

Comment:
@phips28 can you please add the same edit in the bitget.ts file instead?

Reply:
done